docs/source/translations.rst

```diff
@@ -0,0 +1,60 @@
+Translations
+============
+
+.. note::
+    This is only available on Plone 5.
+
+
```

Comment:
@erral a short introduction about Plone's multilingual capabilities would be nice. Keep in mind that the docs should work for JS devs who do not know the internals of Plone. One or two sentences are sufficient.